crates/store/re_chunk_store/src/query.rs

```diff
@@ -41,7 +64,27 @@ impl ChunkStore {
     }
 
     /// Retrieve all [`ComponentName`]s in the store.
-    pub fn all_components(&self) -> BTreeSet<ComponentName> {
+    pub fn all_components(&self) -> IntSet<ComponentName> {
+        self.static_chunk_ids_per_entity
+            .values()
+            .flat_map(|static_chunks_per_component| static_chunks_per_component.keys())
+            .chain(
+                self.temporal_chunk_ids_per_entity_per_component
+                    .values()
+                    .flat_map(|temporal_chunk_ids_per_timeline| {
+                        temporal_chunk_ids_per_timeline.values().flat_map(
+                            |temporal_chunk_ids_per_component| {
+                                temporal_chunk_ids_per_component.keys()
+                            },
+                        )
+                    }),
+            )
+            .copied()
+            .collect()
+    }
+
+    /// Retrieve all [`ComponentName`]s in the store.
+    pub fn all_components_sorted(&self) -> ComponentNameSet {
```

Comment:
Maybe we should rename this type to `OrderedComponentNameSet` to pair it up with `UnorderedComponentNameSet`, a bit confusing in the context of this PR :/

(or even better imho `ComponentNameOrderedSet` & `ComponentNameUnorderedSet` for friendlier autocomplete)

Reply:
I wanted to, but the UI code already exposes a `SortedComponentSet` with slightly different rules... :zany_face:

Reply:
🙈